pkg/graveler/committed/manager.go

```diff
@@ -105,10 +117,10 @@ func (c *committedManager) Merge(ctx context.Context, ns graveler.StorageNamespa
 	defer baseIt.Close()
 	patchIterator := NewMergeIterator(ctx, diffIt, baseIt)
 	defer patchIterator.Close()
-	return c.Apply(ctx, ns, destination, patchIterator)
+	return c.applyOnDiffWithRanges(ctx, ns, destination, patchIterator)
 }
 
-func (c *committedManager) Apply(ctx context.Context, ns graveler.StorageNamespace, rangeID graveler.MetaRangeID, diffs graveler.ValueIterator) (graveler.MetaRangeID, graveler.DiffSummary, error) {
+func (c *committedManager) applyOnDiffWithRanges(ctx context.Context, ns graveler.StorageNamespace, rangeID graveler.MetaRangeID, diffs Iterator) (graveler.MetaRangeID, graveler.DiffSummary, error) {
```

Comment:
Why the change of names?  Is it no longer possible to call this `apply` from the outside?

Reply:
The diff might be confusing because `Apply` logic moved into `applyOnDiffWithRanges`
`Apply` still exists, it now calls `applyOnDiffWithRanges`.
`Apply` gets a diff with no ranges, because graveler is not aware of ranges.